Think I've fixed it.

~~Anyone know what is wrong here?~~
```
          <Link href={`/piece?id=${item._id}`} as={`/piece/${item._id}`}>
            <a
              style={{ textDecoration: 'none', height: '48px' }}
              target="_self"
              onClick={closeDrawer}
            >
              <Thumb src={`/static/uploads/${item.images[0].thumb}`} />
            </a>
          </Link>
```
~~linter says:~~
```
/home/nik/projects/chingu/voyage8/bears14/components/CartDrawer/CartItem.js
  29:13  error  Visible, non-interactive elements with click handlers must have at least one keyboard listener  jsx-a11y/click-events-have-key-events
  29:13  error  Static HTML elements with event handlers require a role                                         jsx-a11y/no-static-element-interactions
```

--- a/components/CartDrawer/CartItem.js
+++ b/components/CartDrawer/CartItem.js
@@ -30,6 +30,9 @@ const CartItem = ({ buyItNow, data, closeDrawer }) => {
               style={{ textDecoration: 'none', height: '48px' }}
               target="_self"
               onClick={closeDrawer}
+              onKeyDown={closeDrawer}
+              role="link"
+              tabIndex={0}
             >
               <Thumb src={`/static/uploads/${item.images[0].thumb}`} />
             </a>

